### DISCUSSION TITLE
How to control top navigation?

### DISCUSSION THREAD

**Discussion by ekansh005** [Budibase/budibase#3888] [Help] [answered]
Hi,
I just started exploring and was creating Vehicles app with internal DB. I once generated a new screen for service logs using 'auto generated screens' option and it started showing that in the top navigation. Then I deleted all three generated screens of service log but it was still showing 'service-log' in the top navigation section. Then I generated these screens again and now I have two 'service-log' entries in the top bar.
![image](https://user-images.githubusercontent.com/439184/147696595-5ab78b16-b6f1-4ec7-b48b-9e0f7074282e.png)
![image](https://user-images.githubusercontent.com/439184/147696636-688c3ce7-cea7-4e43-9f70-18cb43ced819.png)

How does it generate this top navigation bar and how can I control it?

BR, Ekansh

**Comment by shogunpurple** [comment, marked as answer]
Hey @ekansh005 - here's a short video detailing how to control the links in the top navigation bar.

- Go to layouts
- Go to "Navigation Layout"
- Select the "Links" property on the right hand side. This will allow you to delete and control your links for your application.

https://user-images.githubusercontent.com/11256663/147709492-78f8f60a-746c-40e7-bd30-7f26a9918850.mov

.

**Reply by ekansh005** [reply]
Thanks, it worked
